openwisp_network_topology/api/views.py

```diff
@@ -75,7 +75,15 @@ def post(self, request, pk, format=None):
                 'with message "%s"'
             ) % (topology.get_parser_display(), e.__class__.__name__, e)
             return Response({'detail': error}, status=400)
-        return Response({'detail': _('data received successfully')})
+        success_message = 'data received successfully'
+        deprecated_url = f'/api/v1/receive/{pk}/'
+        if request.path == deprecated_url:
+            warning = (
+                ". But this url is deprecated and will soon be removed."
+                " We now use /topology/{id}/receive/"
```

Comment:
Simplify: "This URL is depercated and will be removed in future versions, use /topology/{id}/receive/".

Make translatable at the moment of defining it here as well.
```python
warning = _('This URL is depercated and will be removed in future versions, use /topology/{id}/receive/')
success_message = '{success_message}. {warning}'
```